example/config.yml

```diff
@@ -68,6 +68,7 @@ collections: # A list of collections the CMS should be able to edit
         collection: "posts"
         searchFields: ["title", "body"]
         valueField: "title"
+        displayField: "title"
```

Comment:
`displayField` should default to value of `valueField` so that existing implementations don't break. These examples changes can then be removed.